src/graph.jl

```diff
@@ -3,12 +3,13 @@
 # It implements edge_list, adjacency_list and incidence_list
 #
 
-type GenericGraph{V,E,VList,EList,IncList} <: AbstractGraph{V,E}
+type GenericGraph{V,E,VList,EList,IncList,VDict} <: AbstractGraph{V,E}
     is_directed::Bool
     vertices::VList     # an indexable container of vertices
     edges::EList        # an indexable container of edges
     finclist::IncList   # forward incidence list
     binclist::IncList   # backward incidence list
+    indexof::VDict   # dictionary storing index for each vertex
```

Comment:
I think this should just be ``Dict{V,Int}`` explicitly instead of another type parameter.